layouts/partials/project-navbar.html

```diff
@@ -5,19 +5,21 @@
 <div class="container">
     <div class="row row-centered">            
         <div class="col-xs-6 col-centered col-max">
-            <a href="/{{ .Params.projectSlug }}/">Get Started</a>
+            <a href="/{{ .Params.projectslug }}/">Get Started</a>
         </div>
-        <!--<div class="col-xs-6 col-centered col-max">
-            <a href="/{{ .Params.projectSlug }}/developer-guide/">Developer Guide</a>
-        </div>-->
+        {{ if ne .Params.projectslug "android" }}
```

Comment:
The branch [feature/section-based-templates](https://github.com/NASAWorldWind/NASAWorldWind.github.io/tree/feature/section-based-templates) pulls the project navbar into project/section specific templates. This will eliminate the projectslug, mainpage, and projectpage front matter variables entirely.

Duplication of html and code exists between the templates, but the complexity of the list template is greatly reduced. Take note of the reduction in [front matter](https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/feature/list-ordering-examples/content/android/_index.md) and [archetype metadata](https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/feature/section-based-templates/archetypes/android.md).